DataParTemplate.m

```diff
@@ -319,6 +318,21 @@
 %                       ...
 %                       }]
 %                       (OPTIONAL, DEFAULT={'TotalGM','DeepWM'})
+%                       Available atlases (please check the atlas NIfTI and accompanying files for more information):
```

Comment:
If we do this, we should add which atlases are open source (those in SPMmodified subdirectories and Mindboggle) and free for non-commercial use (e.g., HammersCAT12)

Reply:
Good point. Modified that in the code